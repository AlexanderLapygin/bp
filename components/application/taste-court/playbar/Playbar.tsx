import * as React from 'react'
import {createStyles, Toolbar, Typography} from "@material-ui/core";
import {AppBar} from "@material-ui/core";
import {makeStyles} from "@material-ui/core";

const useStyles = makeStyles(() =>
  createStyles({
    background: {
      background: '#37474F'
    }
  }),
);

export const Playbar = () => {
  const classes = useStyles({});
  return (
    <AppBar position="relative" className={classes.background}> {/* probably not the best practice */}
      <Toolbar>
        <Typography variant="h6">
          Playbar
        </Typography>
      </Toolbar>
    </AppBar>
  )
}
