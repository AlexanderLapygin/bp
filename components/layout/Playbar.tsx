import React from 'react'
import {Toolbar, Typography} from "@material-ui/core";
import {AppBar} from "@material-ui/core";
import {createStyles, makeStyles, Theme} from '@material-ui/core/styles'

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      flexGrow: 1
    },
    menuButton: {
      marginRight: theme.spacing(2),
    },
    title: {
      flexGrow: 1,
    },
    background: {
      background : '#37474F'
    },
    toolbarBackground: {
      background : 'green'
    }
  }),
);

export const Playbar = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <AppBar position="static" className={classes.background}> {/* probably not the best practice */}
        <Toolbar>
          <Typography variant="h6" className={classes.title}>
            Playbar
          </Typography>
        </Toolbar>
      </AppBar>
    </div>
  )
}
