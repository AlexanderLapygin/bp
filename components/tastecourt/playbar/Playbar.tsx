import * as React from 'react'
import {createStyles, Theme, Toolbar, Typography} from "@material-ui/core";
import {AppBar} from "@material-ui/core";
import {makeStyles} from "@material-ui/core";

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      flexGrow: 1
    },
    menuButton: {
      marginRight: theme.spacing(2),
    },
    title: {
      flexGrow: 1,
    },
    background: {
      background : '#37474F'
    },
    toolbarBackground: {
      background : 'green'
    }
  }),
);

export const Playbar = () => {
  const classes = useStyles({});
  return (
    <div className={classes.root}>
      <AppBar position="static" className={classes.background}> {/* probably not the best practice */}
        <Toolbar>
          <Typography variant="h6" className={classes.title}>
            Playbar
          </Typography>
        </Toolbar>
      </AppBar>
    </div>
  )
}
