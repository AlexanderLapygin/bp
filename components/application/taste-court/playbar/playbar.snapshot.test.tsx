import * as React from "react";
import * as TestRenderer from 'react-test-renderer';
import {Playbar} from "./Playbar";

it('renders correctly', () => {
  const tree = TestRenderer.create(<Playbar/>).toJSON()
  expect(tree).toMatchSnapshot()
})
