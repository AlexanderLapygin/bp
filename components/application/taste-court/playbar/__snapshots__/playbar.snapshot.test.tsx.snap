// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative makeStyles-background-1 MuiAppBar-colorPrimary"
>
   
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h6
      className="MuiTypography-root MuiTypography-h6"
    >
      Playbar
    </h6>
  </div>
</header>
`;
